src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -131,11 +132,13 @@
            [contacts
             {:pending-contact-requests pending-contact-requests
```

Comment:
hmm, both of these components are using 1 param of a map. So this should be 
```
  [contacts
            {:pending-contact-requests pending-contact-requests
             :set-scroll-ref           set-scroll-ref
             :scroll-shared-value      scroll-shared-value
             :theme theme}]
           [chats
            {:selected-tab        selected-tab
             :set-scroll-ref      set-scroll-ref
             :scroll-shared-value scroll-shared-value
             :theme theme}
           ])
   ```

Reply:
you are right